[Marketplace Contribution] Akamai WAF - Content Pack Update (#25956)
demisto/content: Packs/Akamai_WAF/ReleaseNotes/2_0_5.md

#### Integrations

##### Akamai WAF

> - %%UPDATE_RN%%

Reviewer: Added the UPDATE RN.

#### Integrations

##### Akamai WAF

> - Updated the command "create_enrollment_command" with the support of multiple Sans.
> - Added a new command "list_siteshield_maps_command" to list all Site Shield maps that belong to your account.
> - Updated the Docker image to: *demisto/akamai:1.0.0.55039*.

Reviewer 2: ```suggestion
- Added support for the *sans* argument in the ***akamai-create-enrollment*** command.
- Added a new command ***akamai-list-siteshield-map*** to list all Site Shield maps that belong to your account.
- Updated the Docker image to: *demisto/akamai:1.0.0.55039*.
```
